<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterGUI</class>
 <widget class="QMainWindow" name="PrinterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>569</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>563</width>
    <height>569</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>563</width>
    <height>569</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="file_path_line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter file path...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GENERATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_path_label">
    <property name="geometry">
     <rect>
      <x>467</x>
      <y>19</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Invalid</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_path_line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter output path...</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_path_label">
    <property name="geometry">
     <rect>
      <x>467</x>
      <y>70</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Invalid</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dither_check_box">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="brightness_spin_box">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string>%</string>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="dither_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dither</string>
    </property>
   </widget>
   <widget class="QLabel" name="brightness_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>% Brightness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_name_line">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>450</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter output name...</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dpi_spring_box">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QLabel" name="dpi_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DPI</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
